eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -233,26 +234,45 @@ isValidAlonzoTxL = lens isValid (\tx valid -> tx {isValid = valid})
 {-# INLINEABLE isValidAlonzoTxL #-}
 
 deriving instance
-  (Era era, Eq (Core.TxBody era), Eq (TxWits era), Eq (AuxiliaryData era)) =>
+  ( Era era,
+    Eq (Core.TxBody era),
+    Eq (TxAuxData era),
+    Eq (TxWits era)
+  ) =>
   Eq (AlonzoTx era)
 
 deriving instance
-  (Era era, Show (Core.TxBody era), Show (TxWits era), Show (AuxiliaryData era)) =>
+  ( Era era,
+    Show (Core.TxBody era),
+    Show (TxAuxData era),
+    Show (Script era),
+    Show (TxWits era)
+  ) =>
   Show (AlonzoTx era)
 
 instance
   ( Era era,
-    NoThunks (AuxiliaryData era),
     NoThunks (TxWits era),
-    NoThunks (Core.TxBody era)
+    NoThunks (TxAuxData era),
+    NoThunks (Script era),
+    NoThunks (Core.TxBody era),
+    NoThunks (Value era),
+    NoThunks (PParamsUpdate era)
```

Comment:
Redundant constraints:
```suggestion
    NoThunks (Core.TxBody era)
```

Reply:
Interesting that the compiler didn't  complain about this ..Compiler 0 - @lehins 1 

And actually not even haskell -language-server does (assuming it works correctly)

Reply:
done